soweego/linker/baseline.py

```diff
@@ -111,7 +111,7 @@ def perfect_name_match(source_dataset, target_entity: BaseEntity, target_pid: st
     for row_entity in source_dataset:
         entity = json.loads(row_entity)
         qid = entity['qid']
-        for label in entity['label'].keys():
+        for label in entity['name']:
```

Comment:
```suggestion
        for label in entity[constants.NAME]:
```
Please avoid hardcoded keys. Remember to add `from soweego.commons import constants`